.travis.yml

```diff
@@ -1,12 +1,18 @@
 language: node_js
 node_js:
   - "stable"
+  - "5.1"
+  - "5"
+  - "4.2"
   - "4.1"
   - "4.0"
   - "0.12"
   - "0.11"
   - "0.10"
   - "iojs"
+matrix:
+  allow_failures:
+    - node_js: "stable"
```

Comment:
Node v6 fails right now:

https://travis-ci.org/dterei/memjs/jobs/126579639

Due to compilation problems on Travis before we even test memjs. I moved it to allow fail for now until someone can be motivated to fix travis.